template/format_index_html.py

```diff
@@ -1,7 +1,7 @@
 import sys
 import fileinput
 
-file = 'templates/index.html'
+file = 'index.html'
```

Comment:
Hm, is there a reason that this was done? This seems to break things as the `{%static }` tags don't get added to the templated `index.html`.

Reply:
@joshm91 does this still persist?

Reply:
I've changed it locally back to `templates/index.html` so it works for me that way.

I'm very new to Vue, JS and webpack so maybe I'm missing something here but as far as I can tell, `npm build` in `deploy.sh` takes a copy of `./index.html`, adds all of the static app js links to it and places it in `./templates/index.html`.

`deploy.sh` then runs `format_index_html.py` which looks at `./index.html` to try and add Django template tags to any static files but none will exist because they're in `./templates/index.html` instead, right?  So surely the `file` variable needs to be set to `templates/index.html` instead.

Thanks for responding!